explorer_frontend/src/features/contracts/components/Deploy/ImportContractTab.tsx

```diff
@@ -94,7 +96,7 @@ export const ImportContractTab = () => {
       <div>
         <Button
           onClick={() => {
-            if (!error) assignSmartContract();
+            if (!error) importSmartContract();
```

Comment:
Why don't we trigger validate on error, catching in UseEffect can create quite a fee side effects. Personally i don't like errors while i am typing, but when i press validate input if it's not valid show me error if it's valid execute action


https://github.com/user-attachments/assets/121c1369-c714-4e58-82e5-f3d6cfb4bc94

Reply:
Yes, that's correct 👍 . There were plenty of issues with this feture.